docs/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: HCP
+meta_desc: Provides an overview of the HashiCorp Cloud Platform Provider for Pulumi.
+layout: overview
+---
+
+The HashiCorp Cloud Platform provider (HCP) for Pulumi can be used to
+provision any of the cloud resources available in [HashiCorp Cloud
+Platform](https://www.hashicorp.com/cloud-platform). The HCP provider
+must be configured with credentials to deploy and update resources.
+
+## Example
```

Comment:
Should we just deprecate this section and tell people to check out examples/ ?

Reply:
This will eventually be what is displayed to people looking at this plugin from the [Pulumi Registry](https://www.pulumi.com/registry/)